src/compiler/utilities.ts

```diff
@@ -4215,6 +4158,16 @@ namespace ts {
         return node.kind === SyntaxKind.ParenthesizedExpression;
     }
 
+    export function skipPartiallyEmittedExpressions(node: Expression): Expression;

```

Comment:
Why was this moved? All of the other related `skip` functions are still in _factory.ts_.